src/app/navbar/navbar.component.html

```diff
@@ -1,8 +1,12 @@
 <header>
   <nav class="navbar">
-    <div (click)="routeToHome()" class="logo-container cursor-pointer">
+    <a
+      [routerLink]="['/home']"
```

Comment:
Since it a static route, consider using routerlink without the square bracket
```suggestion
      routerLink="/home"
```